wallpaper-bindings/src/lib.rs

```diff
@@ -0,0 +1,13 @@
+use more_wallpapers::Mode;
+
+pub fn set_wallpaper_from_file(file_path: &str) {
```

Comment:
We should accept a `&PathBuf` instead of a `&str` here, since all our filesystem stuff in the other crates work with `PathBuf`.